setup.py

```diff
@@ -193,6 +193,23 @@ def unix_can_compile(c_code):
         shutil.rmtree(tempdir)
 
 
+def get_systemd_version():
+    try:
+        r = subprocess.run(["systemctl", "--version"], capture_output=True)
```

Comment:
Yes. It tried this patch.

`capture_output` is present since Python 3.7: https://docs.python.org/3/library/subprocess.html#using-the-subprocess-module

But now that the last condition for systemd disappeared, we can remove this part too.

Reply:
Oh sorry. I didn't realize that the failing CI job was about Python 2.7, where `capture_output` does not exist.